_docker/drupal/drupal-filesystem/web/modules/custom/rhd_assemblies/src/Service/WordpressApi.php

```diff
@@ -256,6 +269,12 @@ private function getContentComposite($content) {
     return $item;
   }
 
+  /**
+   * Please update this function description..
```

Comment:
Formats multiple Wordpress posts, returned from WP JSON API, into a format we want in Drupal